Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -272,6 +272,16 @@ public extension Sequence {
     func filter(by keyPath: KeyPath<Element, Bool>) -> [Element] {
         return filter { $0[keyPath: keyPath] }
     }
+
+    /// SwifterSwift: Returns the first element of the sequence with having property by given key path equal to given `value`.
+    ///
+    /// - Parameters:
+    ///   - keyPath: The `KeyPath` of property for `Element` to compare.
+    ///   - value: The value to compare with `Element` property
+    /// - Returns: The first element of the collection that
+    func first<T: Equatable>(by keyPath: KeyPath<Element, T>, equal value: T) -> Element? {
```

Comment:
I think the actual where reads better in this situation... consider `array1.first(by: \.name, equal: "Tom")` reads `the first element by name equals tom` but I think  `array1.first(where: \.name, equal: "Tom")` reads better : `the first elment where name equals tom` :) 
This is just a suggestion ... I'm not a native speaker so ... let me know what you think

Reply:
@LucianoPAlmeida thanks for your comment. I agree with you. Will update now.